ui/shared/status/StatusChatInput.qml

```diff
@@ -413,24 +437,48 @@ Rectangle {
         messageInputField.forceActiveFocus();
     }
 
+    Connections {
+        target: applicationWindow.dragAndDrop
+        onDroppedOnValidScreen: (drop) => {
+            let validImages = validateImages(applicationWindow.dragAndDrop.droppedUrls)
+            if (validImages.length > 0) {
+                showImageArea(validImages)
+                drop.acceptProposedAction()
+            }
+        }
+    }
+
     ListModel {
         id: suggestions
     }
 
+    Rectangle {
+        id: chatErrorMessage
+        property alias text: txtMessage.text
+        color: Style.current.white
```

Comment:
Should this be `background` as well?

Reply:
Removed the whole component as it wasn't being used 😂